lib/api_bridge/helpers.rb

```diff
@@ -72,6 +72,7 @@ def self.set_page page
   private
 
   def self.encode_query_params(query_string)
+    # puts "QUERY_STRING: #{query_string}"
```

Comment:
:fire: :smile:

Reply:
Yeah, I was meaning to ask you about that...do we want to keep a courtesy debug line commented out for our use?  I'm fine with removing it here and in orchid

Reply:
I'm not opposed to leaving it in there where we can access it easily for debug purposes :)
Maybe add a comment above it that even notes to keep it for that reason?